safe_transaction_service/history/migrations/0068_alter_multisigtransaction_origin.py

```diff
@@ -30,13 +30,12 @@ def repair_backward_migration(apps, schema_editor):
     transactions = MultisigTransaction.objects.filter(origin__isnull=False).iterator()
     for transaction in transactions:
         value = json.loads(transaction.origin)
-        if type(value) == str:
+        if isinstance(value, str):
```

Comment:
This was pre-commit?

Reply:
This was a new problem detected by the new version of flake8